app/views/curation_concerns/base/_form_state.html.erb

```diff
@@ -0,0 +1,13 @@
+<fieldset id="set-state">
+  <legend>State</legend>
+  <div class="form-group">
+    <label class="radio">
```

Comment:
Can we use simple form builders for these? Might push off some of the view branches